__test__/op-connect.test.ts

```diff
@@ -227,6 +227,49 @@ describe("Test OnePasswordConnect CRUD", () => {
             expect(vault.name).toEqual(vaultTitle);
         });
     });
+
+    describe("list items by title", () => {
+        const title = "some title";
+        const op = OnePasswordConnect(testOpts);
+        const getItemsByTitleMock = (title: string) => nock(mockServerUrl)
```

Comment:
This is a really nice code simplification. Do you think you can do the same thing for vaults tests? (we can do this in a separate PR)

Reply:
Yes, we even can extract everything in some `api.mock.ts` module and use it in the tests in this way. Let's do such a refactoring in the other PR :)